ev-dev-tools/src/ev_cli/helpers.py

```diff
@@ -97,6 +97,7 @@ def primitive_to_sample_value(type):
     'object': 'Object',
 }
 
+template_version_pattern = '//\n// AUTO GENERATED - DO NOT EDIT!\n// template version (\d+)'
```

Comment:
As discussed elsewhere, wouldn't it be simpler to rely on the timestamp of the template files?  Especially for development, you don't want to manually increase the template version each time.  Furthermore this is quite brittle, as it depends on the specific string, which could be also changed at some point.